scipy/optimize/tests/test_linprog.py

```diff
@@ -904,6 +910,11 @@ def f():
 class TestLinprogIPSparse(BaseTestLinprogIP):
     options = {"sparse": True}
 
+    @pytest.mark.xfail(reason='Fails with ATLAS, see gh-7877')
+    def test_bug_6690(self):
+        # Test defined in base class, but can't mark as xfail there
+        pass
```

Comment:
Maybe add `super(TestLinprogIPSparse, self).test_bug_6690()` so we get the xpass status when it passes?

Reply:
Good idea, done.